src/nvim/ex_cmds.c

```diff
@@ -4960,3 +4961,56 @@ void ex_oldfiles(exarg_T *eap)
     }
   }
 }
+
+void ex_trust(exarg_T *eap)
+{
+  char *p = skiptowhite(eap->arg);
+  size_t len = (size_t)(p - eap->arg);
+  char *arg1 = xmemdupz(eap->arg, len);
+
+  bool success = false;
+  bool action_allow = false, action_deny = false, action_remove = false;
+  char *msg = NULL;
+
+  if (len > 2 && STRNCMP(arg1, "++deny", len) == 0) {
```

Comment:
Is the `len > 2` check necessary? If `len < 2` then `STRNCMP` will be false anyway.

Reply:
Without that check it'd match `++` as `++deny`.

Reply:
Oh right 🤦‍♂️

Reply:
Is `arg1` guaranteed to be null-terminated? If so we can just use `strcmp` (since `"++deny"` is a static string with known length).

Reply:
Yes, arg1 will be always null terminated

Reply:
Done in 91345bc149d7d5ef65d6c0735cb698ef42c9068c